app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,5 @@
+class DashboardController < ApplicationController
```

Comment:
I would expect to get the list of dashboard in the index action of the `DashboardController`, and It might be better to use the `show` action in case we want to follow this path.